stdlib/public/SDK/simd/simd.swift.gyb

```diff
@@ -68,8 +68,8 @@ public struct ${vectype} {
   public init() { self.init(0) }
 
   @_transparent
-  public init(_ _value: Builtin.${llvm_vectype}) {
```

Comment:
I missed this before, but please don't add an unlabeled initializer if you can at all avoid it. Those things cause exponential behavior in the type checker all over the place.

Reply:
@DougGregor Mark already fixed the immediate issue, but if we want to remove *all* the unlabeled inits from Builtins (Int, Float, simd), I'm happy to make a patch that does that. I just want them to be uniform.